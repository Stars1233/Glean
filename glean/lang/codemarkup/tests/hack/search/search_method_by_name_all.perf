{
  "@generated": null,
  "facts_searched": { "hack.SearchMethodByName.6": 7 },
  "full_scans": [ "hack.SearchMethodByName.6" ]
}