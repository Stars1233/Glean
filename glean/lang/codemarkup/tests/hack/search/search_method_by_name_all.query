query: hack.SearchMethodByName { name = _ }
transform: [normord, []]
perf: true
