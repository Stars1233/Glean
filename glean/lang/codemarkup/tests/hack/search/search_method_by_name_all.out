[
  "@generated",
  {
    "key": {
      "decl": {
        "method": {
          "key": {
            "container": {
              "class_": {
                "key": { "name": { "key": { "name": { "key": "RefClass" } } } }
              }
            },
            "name": { "key": "bar" }
          }
        }
      },
      "name": "bar",
      "parent": { "key": { "name": { "key": "RefClass" } } }
    }
  },
  {
    "key": {
      "decl": {
        "method": {
          "key": {
            "container": {
              "class_": {
                "key": { "name": { "key": { "name": { "key": "RefClass" } } } }
              }
            },
            "name": { "key": "bazza" }
          }
        }
      },
      "name": "bazza",
      "parent": { "key": { "name": { "key": "RefClass" } } }
    }
  },
  {
    "key": {
      "decl": {
        "method": {
          "key": {
            "container": {
              "class_": {
                "key": { "name": { "key": { "name": { "key": "RefClass" } } } }
              }
            },
            "name": { "key": "foo" }
          }
        }
      },
      "name": "foo",
      "parent": { "key": { "name": { "key": "RefClass" } } }
    }
  },
  {
    "key": {
      "decl": {
        "method": {
          "key": {
            "container": {
              "class_": {
                "key": {
                  "name": {
                    "key": {
                      "name": { "key": "SourceClass" },
                      "namespace_": { "key": { "name": { "key": "NS" } } }
                    }
                  }
                }
              }
            },
            "name": { "key": "bar" }
          }
        }
      },
      "name": "bar",
      "parent": {
        "key": {
          "name": { "key": "SourceClass" },
          "namespace_": { "key": { "name": { "key": "NS" } } }
        }
      }
    }
  },
  {
    "key": {
      "decl": {
        "method": {
          "key": {
            "container": {
              "class_": {
                "key": {
                  "name": { "key": { "name": { "key": "SuperClass" } } }
                }
              }
            },
            "name": { "key": "superAdd" }
          }
        }
      },
      "name": "superAdd",
      "parent": { "key": { "name": { "key": "SuperClass" } } }
    }
  },
  {
    "key": {
      "decl": {
        "method": {
          "key": {
            "container": {
              "interface_": {
                "key": {
                  "name": { "key": { "name": { "key": "SourceInterface" } } }
                }
              }
            },
            "name": { "key": "foo" }
          }
        }
      },
      "name": "foo",
      "parent": { "key": { "name": { "key": "SourceInterface" } } }
    }
  },
  {
    "key": {
      "decl": {
        "method": {
          "key": {
            "container": {
              "trait": {
                "key": {
                  "name": { "key": { "name": { "key": "SourceTrait" } } }
                }
              }
            },
            "name": { "key": "quux" }
          }
        }
      },
      "name": "quux",
      "parent": { "key": { "name": { "key": "SourceTrait" } } }
    }
  }
]